administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-06-08.sql

```diff
@@ -24,3 +24,6 @@ CREATE TABLE IF NOT EXISTS `#__cookiemanager_cookies` (
 
 INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`) VALUES
 (0, 'com_cookiemanager', 'component', 'com_cookiemanager', '', 1, 1, 1, 0, 1, '', '{"policylink":"","position":"bottom"}', '');
+
+INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `manifest_cache`, `params`, `custom_data`, `checked_out`, `checked_out_time`, `ordering`, `state`) VALUES
+(0, 'plg_system_cookiemanager', 'plugin', 'cookiemanager', 'system', 0, 1, 1, 0, '', '', '', 0, '0000-00-00 00:00:00', 0, 0);
```

Comment:
As far as I remember I had explained you once that when looking for examples to insert new extensions, you should not take too old scripts as example because they run before the null dates changes.

But you caught such an old example where 0 instead of NULL is used for `checked_out`, and '0000-00-00 00:00:00' instead of NULL is used for `checked_out_time`.

In addition it is just silly to have these 2 insert statements into the same table below each other using different columns in the statement, depending where they have been copied from.

You should instead use the first insert statement to insert both records at once, the one for the component and the other one for the plugin.

So it would be:

```
INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`) VALUES
(0, 'com_cookiemanager', 'component', 'com_cookiemanager', '', 1, 1, 1, 0, 1, '', '{"policylink":"","position":"bottom"}', ''),
(0, 'plg_system_cookiemanager', 'plugin', 'cookiemanager', 'system', 0, 1, 1, 0, 1, '', '', '');
```

Reply:
Thanks and sorry I forgot very quickly